common/utils/string.utils.ts

```diff
@@ -1,6 +1,7 @@
 import replaceSpecialCharacters from "replace-special-characters";
 
-export const labelToUrlComponent = (label: string): string => {
+export const labelToUrlComponent = (label?: string): string | undefined => {
```

Comment:
Why add this complexity here and not in the functions which use this function?

Reply:
Fixed :D